src/app/components_ngrx/work-item-quick-add/work-item-quick-add.component.ts

```diff
@@ -86,7 +93,32 @@ export class WorkItemQuickAddComponent implements OnInit, OnDestroy, AfterViewIn
         .subscribe(items => {
           // const addedItem = items.find(item => item.createId === this.createId);
           this.resetQuickAdd();
-        })
+        }),
+      // This is temporary as the permissions model is not yet ready, currently the
+      // non-collaborator will only be able to add workitems in Openshift_io space
+      this.spaceQuery.getCurrentSpace
```

Comment:
Can we combine these two queries? Can we create a new query for this permission? This doesn't look like the best solution.